topydo.conf

```diff
@@ -6,6 +6,7 @@ default_command             = ls
 ; archive_filename          = done.txt
 colors                      = 1
 identifiers                 = linenumber ; or: text
+; backup                    = 1
```

Comment:
I would combine the two new options. If `backup` is set, then make backups. If `0` or empty, don't.